clients/python/llmengine/completion.py

```diff
@@ -33,6 +33,10 @@ async def acreate(
         temperature: float = 0.2,
         stop_sequences: Optional[List[str]] = None,
         return_token_log_probs: Optional[bool] = False,
+        presence_penalty: float = 0.0,  # vllm, lightllm
```

Comment:
can you make these optional? is there a reason for them to be not optional?

Reply:
also no need to comment here about frameworks since it's specific later in main comment